ui/src/message_popup/transformations.jsx

```diff
@@ -13,13 +12,11 @@ export function withStudents(questions) {
   });
 }
 
-export function withLearningObjectiveAndIndicator(question) {
-  const learningObjective = _.find(learningObjectives, { id: question.learningObjectiveId });
+export function withIndicator(question) {
   const indicator = _.find(indicators, { id: question.indicatorId });
   return {
     ...question,
-    indicator,
-    learningObjective
+    indicator
   };
 }
 
```

Comment:
Now that this exists, can the inside of `questionsForIndicator` be replaced with `withIndicator`?